container-core/src/main/java/com/yahoo/container/handler/LogReader.java

```diff
@@ -122,6 +124,7 @@ public LineWithTimestamp next() {
         @Override
         public void close() throws IOException {
             reader.close();
+            inputStream.close();
```

Comment:
I don't think this is necessary though as closing a `BufferedReader` also closes the underlying reader.

Reply:
Yes, you are right. Closing PR